unreleased/loki-deprecate-format.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
This changes the type of an exported field in an exported struct, and is thus a breaking change.

Reply:
That's true. I'll mark it as such.